.changeset/cyan-lions-mate.md

```diff
@@ -0,0 +1,9 @@
+---
+'@verdaccio/logger-commons': patch
+'@verdaccio/server': patch
+'@verdaccio/logger': patch
```

Comment:
Changing `setup` to return a promise is a breaking change, so I'm not sure if `patch` is correct here. Let me know what you think.